refinery/tool_manager/views.py

```diff
@@ -191,7 +191,8 @@ def relaunch(self, request, *args, **kwargs):
             return HttpResponseBadRequest("Can't relaunch a Tool that is "
                                           "currently running")
         try:
-            return visualization_tool.launch()
+            visualization_tool.launch()
+            return JsonResponse(ToolSerializer(visualization_tool).data)
```

Comment:
Maybe we should keep this consistent with the above:
```
serializer = ToolSerializer(tool)
return JsonResponse(serializer.data)
```

or vice-versa

Reply:
Which do you prefer?

Reply:
The bottom is easier to read.